akd/src/node_state.rs

```diff
@@ -148,6 +148,31 @@ impl NodeLabel {
         Self::new(out_val, len)
     }
 
+    pub(crate) fn get_sibling_prefix(&self, mut len: u32) -> Self {
```

Comment:
nit: Comments explaining what is going on here would be probably good for future support.